modules/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bitgo",
-  "version": "9.1.0",
+  "version": "9.2.0-rc.0",
```

Comment:
Apparently, the SDK publishing process is quite different 
1) We do not manually update the SDK version and leave it to the "yarn publish" to update during a publish. Please revert this change.

2) Manual changes to yarn.lock are not required as it will get updated automatically by the "yarn update" or "yarn publish" process.

With that said, we should have this PR against the release branch, rel/naples, instead of master.

Reply:
> Manual changes to yarn.lock are not required as it will get updated automatically by the "yarn update" or "yarn publish" process.

Actually, if you are making changes to the package.json, please update the yarn.lock file. The package-lock.json in modules/express will  be taken care of for you, do not modify that one.